go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/roc-project/roc-go
```

Comment:
Shouldn't we place this file into `roc` and rename the module to `github.com/roc-project/roc-go/roc`?

Reply:
I'm not sure why do want to do this? do we plan to have more folders in `github.com/roc-project/roc-go/` with their own `go.mod`s? or is it just for the sake of changing the module name to `roc`?

Reply:
My bad, my understanding of go modules was wrong. I though that one module == one package, but that's not true. Let's keep it as is.

Reply:
alright!